cli/cli.go

```diff
@@ -326,12 +326,12 @@ func (c *CLI) getNextMarker(sPos StreamPos, resp *rds.DownloadDBLogFilePortionOu
 	curMin, _ := strconv.Atoi(now.Format("04"))
 	if curMin > 5 {
 		logrus.WithField("newMarker", *resp.Marker).
-			Debugf("no log data received but it's %d minutes (> 5) past " +
+			Debug("no log data received but it's %d minutes (> 5) past " +
```

Comment:
it was a bug before, but now it's a different bug. `%d` should have had an argument before and shouldn't be in the function that doesn't take arguments. ;)